homeassistant/components/zwave/__init__.py

```diff
@@ -310,6 +317,17 @@ def log_all(signal, value=None):
 
         dispatcher.connect(log_all, weak=False)
 
+    @websocket_api.async_response
+    async def websocket_network_status(hass, connection, msg):
```

Comment:
Why define this inside the `async_setup_entry`? It should just use `hass.data[DATA_NETWORK]` and then be defined outside of it.